aspnetcore/signalr/messagepackhubprotocol.md

```diff
@@ -81,18 +81,6 @@ After installing the npm package, the module can be used directly via a JavaScri
 
 *node_modules\\@microsoft\signalr-protocol-msgpack\dist\browser\signalr-protocol-msgpack.js* 
 
-In a browser, the `msgpack5` library must be referenced. Use a `<script>` tag to create a reference. The library can be found at *node_modules\msgpack5\dist\msgpack5.js*.
-
-The following javaScript files must be referenced in the order shown below:
-
-
-```html
-<script src="~/lib/signalr/signalr.js"></script>
-<script src="~/lib/msgpack5/msgpack5.js"></script>
-<script src="~/lib/signalr/signalr-protocol-msgpack.js"></script>
```

Comment:
Should probably still keep part of this to show how to reference the library.
Just signalr.js and signalr-protocol-msgpack.js are needed